### PR TITLE
Feature/viite 2568 AdministrativeClass addition, UI

### PR DESCRIPTION
**Pull request by marjukka-kokkonen-sitowise**

### CODE REVIEW

File: viite-UI/src/model/ApplicationModel.js [hunk 1]
```diff
@@ -259,10 +259,10 @@
...
       },
       assetDragDelay: 100,
-      setRoadTypeShown: function (bool) {
-        if (roadTypeShown !== bool) {
-          roadTypeShown = bool;
-          eventbus.trigger('road-type:selected', roadTypeShown);
+      setAdministrativeClassShown: function (bool) {
+        if (administrativeClassShown !== bool) {
+          administrativeClassShown = bool;
+          eventbus.trigger('road-type:selected', administrativeClassShown);
```
Comment (AnttiJokela) [inline viite-UI/src/model/ApplicationModel.js:265]: road-type:selected -> administrative-class:selected ?

File: viite-UI/src/view/RoadLayer.js [hunk 2]
```diff
@@ -67,38 +67,38 @@
...
-          roadType = LinkValues.RoadTypeShort.PublicRoad.description;
+    var displayAdministrativeClass = function (administrativeClass) {
+      var administrativeClass;
+      switch (administrativeClass) {
+        case LinkValues.AdministrativeClassShort.PublicRoadClass.value:
+          administrativeClass = LinkValues.AdministrativeClassShort.PublicRoadClass.description;
           break;
-        case LinkValues.RoadTypeShort.FerryRoad.value:
-          roadType = LinkValues.RoadTypeShort.FerryRoad.description;
+        case LinkValues.AdministrativeClassShort.FerryRoad.value:
```
Comment (AnttiJokela) [inline viite-UI/src/view/RoadLayer.js:83]: This does not exist

File: viite-UI/src/view/RoadLayer.js [hunk 3]
```diff
@@ -67,38 +67,38 @@
...
+          administrativeClass = LinkValues.AdministrativeClassShort.FerryRoad.description;
           break;
-        case LinkValues.RoadTypeShort.MunicipalityStreetRoad.value:
-          roadType = LinkValues.RoadTypeShort.MunicipalityStreetRoad.description;
+        case LinkValues.AdministrativeClassShort.MunicipalityStreetClass.value:
+          administrativeClass = LinkValues.AdministrativeClassShort.MunicipalityStreetClass.description;
           break;
-        case LinkValues.RoadTypeShort.PublicUnderConstructionRoad.value:
-          roadType = LinkValues.RoadTypeShort.PublicUnderConstructionRoad.description;
+        case LinkValues.AdministrativeClassShort.PublicUnderConstructionRoad.value:
```
Comment (AnttiJokela) [inline viite-UI/src/view/RoadLayer.js:89]: This does not exist

File: viite-UI/src/view/RoadLayer.js [hunk 4]
```diff
@@ -67,38 +67,38 @@
...
+          administrativeClass = LinkValues.AdministrativeClassShort.PublicUnderConstructionRoad.description;
           break;
-        case LinkValues.RoadTypeShort.PrivateRoadType.value:
-          roadType = LinkValues.RoadTypeShort.PrivateRoadType.description;
+        case LinkValues.AdministrativeClassShort.PrivateRoadClass.value:
+          administrativeClass = LinkValues.AdministrativeClassShort.PrivateRoadClass.description;
           break;
-        case roadType = LinkValues.RoadTypeShort.UnknownOwnerRoad.value:
-          roadType = LinkValues.RoadTypeShort.UnknownOwnerRoad.description;
+        case administrativeClass = LinkValues.AdministrativeClassShort.UnknownOwnerRoad.value:
```
Comment (AnttiJokela) [inline viite-UI/src/view/RoadLayer.js:95]: This is just wrong.. and does not exist

File: viite-UI/src/view/RoadLayer.js (hunk 2 above)
Comment (marjukka-kokkonen-sitowise) [reply viite-UI/src/view/RoadLayer.js:83]: Yep, so do not the other two, a few lines downward 👍

File: viite-UI/src/model/ApplicationModel.js [hunk 5]
```diff
@@ -262,7 +262,7 @@
       setAdministrativeClassShown: function (bool) {
         if (administrativeClassShown !== bool) {
           administrativeClassShown = bool;
-          eventbus.trigger('road-type:selected', administrativeClassShown);
+          eventbus.trigger('administrative-class:selected', administrativeClassShown);
```
Comment (AnttiJokela) [inline viite-UI/src/model/ApplicationModel.js:265]: I assume that this event is handled somewhere and needs to be changed as well?

**Comment by marjukka-kokkonen-sitowise** [comment]
These were/are finally done through another branch. Closing.

**Pull request closed without merging**